bin/init.bash

```diff
@@ -396,6 +411,38 @@ set_harbor_config() {
     replace_set_me "${file}" ".harbor.persistence.disableRedirect" "${disable_redirect}"
 }
 
+# Usage: set_cluster_dns <config-file>
+set_cluster_dns() {
```

Comment:
This does more than just configuring DNS. Could it be named something more generic to setting "cluster provisioner specific configuration options" or could it be split into `set_cluster_dns` and `set_cluster_api`?